omp_player.inc

```diff
@@ -1456,7 +1456,7 @@ native bool:ApplyAnimation(playerid, const animationLibrary[], const animationNa
  * <returns><b><c>0</c></b> if there is no animation applied.</returns>
  */
 native GetPlayerAnimationIndex(playerid);
-
+native bool:IsValidAnimationLibrary(const name[]);
```

Comment:
```suggestion

/**
 * <library>omp_player</library>
 * <summary>Checks if the given animation library is valid.</summary>
 * <param name="name">The animation library name to check.</param>
 * <seealso name="ApplyAnimation" />
 * <returns>true if the animation library is valid, otherwise false.</returns>
 */
native bool:IsValidAnimationLibrary(const name[]);

```

Reply:
I'm not sure if it should be in omp_player.inc.
omp_core.inc is probably better.